HA.md

```diff
@@ -0,0 +1,21 @@
+# High Availability (HA)
+## Introduction
+When we create kubernetes cluster using kops we are able to create a multiple K8S masters and by doing so we can assure that in a case of 
+a master instance failure our K8S functionality won't damaged. 
+In scenario of a single master instance that fails our minions/pods will keep running but there will be no new pods schedualing, kubectl
+won't work (since the api-server will be down).
+
+
+## Kops HA
+We can create HA cluster using kops only in the first creation of the cluster by using the "--master-zones" flag
+example: https://github.com/kubernetes/kops/blob/master/docs/advanced_create.md
+K8S relay on a key value DB named "etcd", which is using the the Qurom concept that the cluster need at least 51% of nodes to be
```

Comment:
> K8S relay on a key value DB named "etcd", which is using the the Qurom concept that the cluster need at least 51% of nodes to be

How about:

K8S relies on a key-value DB named etcd, which is user a Quorum clustering concept.  Three masters are created, each with an etcd database instance.